test/unit/test_model_output.py

```diff
@@ -106,81 +23,68 @@ def test_new_instance():
+        assert mo.model_id == expected_model_id
 
 
 @pytest.mark.parametrize(
-    "file_uri, expected_round_id, expected_model_id",
-    [
-        ("mock:raw/prefix/2420-01-01-team-model.csv", "2420-01-01", "team-model"),
-        ("mock:raw/prefix/2420-01-01-----team-model.parquet", "2420-01-01", "team-model"),
-        ("mock:raw/prefix/2420-01-01____teammodelallonestring.csv", "2420-01-01", "teammodelallonestring"),
-        ("mock:raw/prefix/2420-01-01____look-at-all-the-hyphens-.csv", "2420-01-01", "look-at-all-the-hyphens-"),
-    ],
-)
-def test_parse_file(file_uri, expected_round_id, expected_model_id):
-    mo = ModelOutputHandler(file_uri, "mock:/output-uri")
-    assert mo.round_id == expected_round_id
-    assert mo.model_id == expected_model_id
-
-
-@pytest.mark.parametrize(
-    "input_uri, output_uri, expected_input_file, expected_output_path, expected_file_name, expected_model_id",
+    "mo_path_str, expected_input_file, expected_output_path, expected_file_name, expected_model_id",
     [
         (
-            "mock:bucket123/raw/prefix1/prefix 2/2420-01-01-team-model name with spaces.csv",
-            "mock:bucket123/prefix1/prefix 2",
-            "bucket123/raw/prefix1/prefix 2/2420-01-01-team-model name with spaces.csv",
-            "bucket123/prefix1/prefix 2",
+            "raw/prefix1/prefix 2/2420-01-01-team-model name with spaces.csv",
+            "raw/prefix1/prefix 2/2420-01-01-team-model name with spaces.csv",
+            "prefix1/prefix 2",
             "2420-01-01-team-model name with spaces",
             "team-model name with spaces",
         ),
         (
-            "mock:bucket1.2.3/raw/prefix1/prefix 2/2420-01-01-team-model.name.csv",
-            "mock:bucket123/prefix1/~prefix 2",
-            "bucket1.2.3/raw/prefix1/prefix 2/2420-01-01-team-model.name.csv",
-            "bucket123/prefix1/~prefix 2",
+            "raw/prefix1/~prefix 2/2420-01-01-team-model.name.pqt",
+            "raw/prefix1/~prefix 2/2420-01-01-team-model.name.pqt",
+            "prefix1/~prefix 2",
             "2420-01-01-team-model.name",
             "team-model.name",
         ),
         (
-            "mock:raw/prefix 1/prefix2/2420-01-01-spáces at end .csv",
-            "mock:prefix 1/prefix2",
-            "raw/prefix 1/prefix2/2420-01-01-spáces at end.csv",
-            "prefix 1/prefix2",
+            "raw/raw/prefix 1/prefix2/2420-01-01-spáces at end .csv",
+            "raw/raw/prefix 1/prefix2/2420-01-01-spáces at end.csv",
+            "raw/prefix 1/prefix2",
             "2420-01-01-spáces at end",
             "spáces at end",
         ),
         (
-            "mock:a space/prefix 1/prefix2/2420-01-01 look ma no hyphens.csv",
-            "mock:prefix 1/prefix 🐍",
-            "a space/prefix 1/prefix2/2420-01-01 look ma no hyphens.csv",
+            "raw/prefix 1/prefix 🐍/2420-01-01 look ma no hyphens.csv",
+            "raw/prefix 1/prefix 🐍/2420-01-01 look ma no hyphens.csv",
```

Comment:
"we'll let you put snakes in your file paths, just don't end the file name with a space!!"